### ISSUE TITLE
type WARNING when crafting leather items

### ISSUE THREAD

**Issue by Amadeus-** [arkayenro/arkinventory#847]
I'm getting the following message when I craft leather items:
```
You create [Stonehide Leather Barding]
ArkInventory:  WARNING> bag[3] in location 1 [Bag] type is unknown, queuing for rescan
```
If I then check my inventory, all of the bag slots for the bag is outlined in red and the item I crafted isn't anywhere.   However, if I /reloadui at this point, the inventory looks good again and I see the item in my inventory.

It seems to be happening regardless of what I craft.

**Comment by arkayenro** [comment]
what type of bags do you have in each slot?

**Comment by Amadeus-** [comment]
All of my bags are "Hexweave Bags".

I also want to reiterate that this only happens when creating items via crafting.  (I'm not sure if it happens with all crafting, as I've only tested with with this leatherworker character.)    I've played quite a bit since Legion launched with this character and have not had it happen any other time.

**Comment by Archov** [comment]
I'd like to +1 this, I have full hexweave bags and it's not just stonehide leather barding but all of my LW and even cooking doesn't show until I reload ui.

**Comment by arkayenro** [comment]
before you create something do this;
open the config and go to the workarounds section, in there disable yielding
then create something
if there was an error being generated it should now show up

**Comment by Amadeus-** [comment]
I assume you meant "In-Combat Yielding", so I disabled that.   (I'm not quite sure how this has anything to do with this error?)

This time, when I crafted an item (again, the "Stonehide Leather Barding" as shown in my original posting), there is no warning/error message; however, it does not show up in my inventory at all.   Rather, what I see is a "blank" spot in my inventory that is glowing like a slot does when it contains a new item.

Now, at this point I hit the "restack" button and then the "refresh" button.    This did make the item show up properly; however, it also caused quite a few slots to have a red outline and also made 4 WARNING messages show up in the chat window:
```
WARNING> bag[2] in location 1 [Bag] type is unknown, queuing for rescan.
WARNING> bag[3] in location 1 [Bag] type is unknown, queuing for rescan.
WARNING> bag[4] in location 1 [Bag] type is unknown, queuing for rescan.
WARNING> bag[5] in location 1 [Bag] type is unknown, queuing for rescan.
```
"Restacking" and "Refreshing" does nothing at this point to fix the slots with red outlines; however, doing a /reloadui does fix everything back to normal (at least, until crafting again.)

**Comment by arkayenro** [comment]
the coroutine yielding can suppress errors, and while it says 'in-combat' its actually used all the time.  i was hoping youd get an actual error

can you shuffle your bags around and see if it still happens?  i can craft (alchemy) and not get any issues and i have the 30 slot hexweave bags

can you also try with just ai running, no other mods, to rule out a conflict

**Comment by Amadeus-** [comment]
Yes, it does appear to be a conflict with an addon.    I tried it without any other addons and it worked fine, and then when I enabled my addons again, it did the same thing as before (item slot was blank, etc.)

Is there a debugging addon or something where I can see what all is going on with everything?  (Perhaps something that monitors events, etc.?)    Otherwise, trying to figure out what's conflicting is going to take me forever ...lol.

**Comment by arkayenro** [comment]
try elvui (do you have that?)  theres supposed to be a couple of bag sorting options in it that could be causing the conflict

**Comment by Amadeus-** [comment]
Yea, I use ElvUI as my ui addon.    I'll check (but I have been using ElvUI for a long time.)
